sway/server.c

```diff
@@ -170,10 +170,22 @@ void server_fini(struct sway_server *server) {
 }
 
 bool server_start(struct sway_server *server) {
-	// TODO: configurable cursor theme and size
+	/* This ensures we have a configuration for the default seat */
+	input_manager_verify_fallback_seat();
+
```

Comment:
@emersion looking for feedback on this; I doubt this is the best way to do it...

Background: at this point, there hasn't yet been an explicit `seat0` conf created, only the `*` is available (if you configured the cursor theme with `seat * xcursor_theme ...`).

Thus, the `seat_get_config()` call below return `NULL` without the call to `input_manager_verify_fallback_seat()`.

Note: I picked this particular call by breaking in gdb when the `seat0` conf was created, and checking the call path...

Reply:
I chose another approach, where `server_start()` only sets the default theme. When the default seat's xcursor theme is configured, we export the environment variables there instead (and update XWayland).